microservicecloud/pom.xml

```diff
@@ -127,24 +127,24 @@
 		<profile>
 			<id>staging</id>
 			<build>
-				<plugins>
-					<plugin>
-						<groupId>com.google.cloud.tools</groupId>
-						<artifactId>jib-maven-plugin</artifactId>
-						<version>2.5.2</version>
-						<executions>
-							<execution>
-								<phase>package</phase>
-								<goals>
-									<goal>build</goal>
-									<!-- <goal>dockerBuild</goal> -->
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-
-
-				</plugins>
+				<pluginManagement>
```

Comment:
Hi sun shun, i think we do not need this now. the issue which i was facing earlier, is fixed, i removed the jib plugin from parent pom as it was not needed here.